builder/llpcBuilder.h

```diff
@@ -252,6 +262,84 @@ class Builder : public IRBuilder<>
         Value*        pDivisor,             // [in] Divisor value
         const Twine&  instName = "") = 0;   // [in] Name to give instruction(s)
 
+    // Create a "tan" operation for a scalar or vector float or half.
+    virtual Value* CreateTan(
+        Value*        pInValue,             // [in] Input value
+        const Twine&  instName = "") = 0;   // [in] Name to give instruction(s)
+
+    // Create an "asin" operation for a scalar or vector float or half.
+    virtual Value* CreateASin(
+        Value*        pInValue,             // [in] Input value
```

Comment:
pInValue -> pX
// [in] Input value -> // [in] Input value X
Apply this to all following functions that take only one parameter